Scenarios/Services/PokemonService.cs

```diff
@@ -107,5 +107,15 @@ public async Task<PokemonData> GetPokemonManualBufferedAsync()
                 return serializer.Deserialize<PokemonData>(reader);
             }
         }
+
+        public async Task<PokemonData> GetPokemonAsyncNewJson()
+        {
+            using var response = await _client.GetAsync(_url, HttpCompletionOption.ResponseHeadersRead);
+
+            // Get the response stream
+            var responseStream = await response.Content.ReadAsStreamAsync();
```

Comment:
Curious if this should be`using var` ?

Reply:
@epignosisx This is the new `using declaration` [in C# 8](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations).

`HttpClient.GetAsync` returns a `HttpResponseMessage`, which is `IDisposable`.

The compiler will generate the dispose at the end of the scope of `response`. (in this case the method)

Reply:
Yea I should dispose this as well